### ISSUE TITLE
Test relationship select box in Content Search

### ISSUE THREAD

**Issue by nollymar** [dotCMS/core#17663]
### Describe the bug
When a user types something on a relationship select box and try to pick an option from the list, a javascript error is shown and the option is not selected

```
VM2123:123 Uncaught Error: dojo.data.ItemFileReadStore: Invalid item argument.
    at Object._assertIsItem (eval at <anonymous> (eval at <anonymous> (dojo.js?b=5.2.1:2)), <anonymous>:123:10)
    at Object.getValues (eval at <anonymous> (eval at <anonymous> (dojo.js?b=5.2.1:2)), <anonymous>:151:8)
    at Object.getValue (eval at <anonymous> (eval at <anonymous> (dojo.js?b=5.2.1:2)), <anonymous>:142:21)
    at Object._announceOption (build.js?b=5.2.1:2)
    at Object.advice (dojo.js?b=5.2.1:2)
    at Object.<anonymous> (dojo.js?b=5.2.1:2)
    at Object.eval (eval at <anonymous> (prototype.js:212), <anonymous>:32:52)
    at Object.advice (dojo.js?b=5.2.1:2)
    at Object._25e [as _announceOption] (dojo.js?
```

**This error occurs on version: 5.2.1. On master branch it is working correctly**

### Expected behavior
When a user types something on a relationship select box and picks an option from the list, the content search list should be updated accordingly

### Video

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/8741395/69743992-5d8d2680-110d-11ea-97a0-35569aab6e47.gif)

## Acceptance Criteria
1. Functional test

**Comment by bryanboza** [comment]
Fixed, tested in the last master // Postgres // FF

**Comment by bryanboza** [comment]
Fixed, unable to reproduce on release-5.2.2 // Postgres // FF
